src/include/network/summary.rb

```diff
@@ -91,35 +91,5 @@ def OverviewTableContents(caption, header, contents, first)
         HSpacing(1.5)
       )
     end
-
-    # Create table widget for the overview screens
-    # @param [Yast::Term] header table header
-    # @param [Array] contents table contents
-    # @return table widget
-    def OverviewTable(header, contents)
-      header = deep_copy(header)
-      contents = deep_copy(contents)
-      VBox(VSpacing(0.5), OverviewTableContents("", header, contents, true))
```

Comment:
Yay for removed code!
Even better, you can remove also the above-defined `OverviewTableContents`.